examples/inverse/plot_dics_source_power.py

```diff
@@ -8,19 +8,23 @@
 Compute a Dynamic Imaging of Coherent Sources (DICS) [1]_ filter from
 single-trial activity to estimate source power across a frequency band. This
 example demonstrates how to source localize the event-related synchronization
-(ERS) of beta band activity in the "somato" dataset.
+(ERS) of beta band activity in the
+`somatosensory dataset <https://mne-tools.github.io/stable/manual/datasets_index.html?#somatosensory>`_
```

Comment:
you should not link to absolute html links but link to sphinx anchors.
Also https://mne-tools.github.io is not the main documentation page (it's martinos.org/mne)